source/monitoring/logging.rst

```diff
@@ -0,0 +1,94 @@
+.. _logging:
+
+Logging
+=======
+
+This page provides an overview of the locations Open OnDemand writes logs and useful session data and 
+provides context around these locations.
+
+In general administrators will want to note whether they need to see:
+
+- System type logs for authentication to the :ref:`web-node <glossary>` itself or OOD's own configuration.
+- Session data for things like connecting to interactive apps (Jupyter, Rstudio, Codeserver, etc.) from 
+  the user's PUN, which is already running on the web-node.
+
+System Logs
+-----------
+These logs will provide information around authentication and internal Open OnDemand errors. They are very useful 
+when initially installing and configuring Open OnDemand.
+
+**Apache Logs**
+
+For logs relating to issues about: 
+
+ - Authenticating to the web-node
+ - Configuring OnDemand 
+
+Look in either of:
+
+:file:`/var/log/httpd/<hostname>_error.log`
+
+- OnDemand will log to this location for its own configuration errors.
+
+:file:`/var/log/httpd/<hostname>_access.log`
+
+- Where OnDemand will log succsseful logins.
+
+.. warning::
+
+    There are no entries for failed logins.
+
+**NGINX Logs**
+
+For logs relating to issues about:
+
+ - Users trying to connect to a :ref:`login-node <glossary>` from their PUN check:
+
```

Comment:
I would say first that the nginx logs are the output of the PUN.

I don't think that related issue is right (you're referencing the shell app? it doesn't log much).

Related issues are things specific to users (not site wide).  I can only think of this one
- Debugging issues related to job submission. Specifically checking commands being issued to the scheduler (sbatch, qsub, etc) by searching for `execve`.

Reply:
edited the comment above.

Reply:
I reworded the section to try and capture this as best I could. Let me know what you think.